src/Core/Services/Implementations/UserService.cs

```diff
@@ -586,6 +596,31 @@ await _stripeSyncService.UpdateCustomerEmailAddress(user.GatewayCustomerId,
         return IdentityResult.Success;
     }
 
+    private async Task<IdentityResult> ValidateManagedUserAsync(User user, string newEmail)
+    {
+        var managingOrganizations = await GetOrganizationsManagingUserAsync(user.Id);
```

Comment:
Btw there is a feature flag check in `GetOrganizationsManagingUserAsync`, so it will handle the feature flag for us. This follows the same pattern as `IsManagedByAnyOrganizationAsync`.